lib/curl_addrinfo.c

```diff
@@ -50,6 +50,10 @@
 #  define in_addr_t unsigned long
 #endif
 
+#if defined(WINAPI_FAMILY) && (WINAPI_FAMILY == WINAPI_FAMILY_APP)
+#  include <afunix.h>
```

Comment:
I have never heard of that header before. A google search doesn't lead me to any official microsoft pages. Are you sure this is contained in stock Windows and if so since which version? This must also be checked!

Reply:
It's weird - about the only information about this header is on the MS blog: https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/

Reply:
On my machine this header is located like this:

`C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\shared`

It is bundled with `Windows 10 SDK (10.0.19041.0)`, standard installation via Visual Studio. Check more details here: https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/

This header is also mentioned in the sources too: https://github.com/curl/curl/blob/70cf50fb4aa5e55ca2e732716a4f455d39192088/lib/config-win32.h#L723

Reply:
> It is bundled with Windows 10 SDK (10.0.19041.0), standard installation via Visual Studio. Check more details here: https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/

Then please check for that build and don't discriminate users of older versions of Windows

Reply:
@emilengler, would you please elaborate on your comment?

Even though the SDK is one of the latest you can target the required min. Windows version for the build, in my case project is targeting `10.0.15063.0` which is fairly old 1703 Creators Update from early 2017 (https://en.wikipedia.org/wiki/Windows_10_version_history).

Without this correction curl can not be build successfully as explained in the PR description.

Reply:
@dmitrykos I have somehow missed the last sentence of your PR description apparently- sorry. In that case I think it's a good idea to document that somewhere still